packages/lisk-constants/test/tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+	"extends": ["../../../tslint.test.json"]
```

Comment:
this file could be a template? we can also create a symlink for the root folder.